services/app-web/src/pages/Wrapper/AuthenticatedRouteWrapper.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { Modal } from '../../components/Modal/Modal'
+import { ModalRef, Alert } from '@trussworks/react-uswds'
+import { createRef, useCallback, useEffect } from 'react'
+import { useAuth } from '../../contexts/AuthContext'
+import { AuthModeType } from '../../common-code/config'
+import { extendSession } from '../Auth/cognitoAuth'
+import styles from '../StateSubmission/ReviewSubmit/ReviewSubmit.module.scss'
+import { dayjs } from '../../common-code/dateHelpers/dayjs'
+import { useLDClient } from 'launchdarkly-react-client-sdk'
+import { featureFlags } from '../../common-code/featureFlags'
+
+export const AuthenticatedRouteWrapper = ({
+    children,
+    setAlert,
+    authMode,
+}: {
+    children: React.ReactNode
+    setAlert?: React.Dispatch<React.ReactElement>
+    authMode: AuthModeType
+}): React.ReactElement => {
+    const { logout, isSessionExpiring, timeUntilLogout, updateSessionExpiry } =
+        useAuth()
+    const ldClient = useLDClient()
+    const minutesUntilExpiration = ldClient?.variation(
+        featureFlags.MINUTES_UNTIL_SESSION_EXPIRES,
+        30
+    )
+    const modalRef = createRef<ModalRef>()
+    const LocalStorage = window.localStorage
+    const logoutSession = useCallback(() => {
+        updateSessionExpiry(false)
+        LocalStorage.removeItem('LOGOUT_TIMER')
+        if (logout) {
+            logout().catch((e) => {
+                console.log('Error with logout: ', e)
+                setAlert &&
+                    setAlert(
+                        <Alert
+                            data-testid="Error400"
+                            style={{ width: '600px', marginBottom: '5px' }}
+                            type="error"
+                            heading="Oops! Something went wrong"
+                        />
+                    )
+            })
+        }
+    }, [logout, setAlert, updateSessionExpiry, LocalStorage])
+
+    const resetSessionTimeout = () => {
+        updateSessionExpiry(false)
+        const sessionExpirationTime = dayjs(Date.now()).add(
+            minutesUntilExpiration,
+            'minute'
+        )
+        try {
+            LocalStorage.setItem(
+                'LOGOUT_TIMER',
+                sessionExpirationTime.toISOString()
+            )
+        } catch (e) {
+            console.log('Error setting logout timer: ', e)
+        }
+        if (authMode !== 'LOCAL') {
+            void extendSession()
+        }
+    }
+
+    useEffect(() => {
+        modalRef.current?.toggleModal(undefined, isSessionExpiring)
+    }, [isSessionExpiring, modalRef])
+
+    useEffect(() => {
+        if (timeUntilLogout < 1) {
+            logoutSession()
+        }
+    }, [timeUntilLogout, logoutSession])
+    return (
+        <div className={styles.wrapper}>
```

Comment:
This wrapper div is a bit problematic. Looks like its messing up our full screen styles for content inside `<main />`. See images below.

 Could you just use a React fragment here instead? Not sure we need the nested div here or its styles at all...
![Screen Shot 2022-05-20 at 2 40 25 PM](https://user-images.githubusercontent.com/10750442/169611373-171bc142-ab02-48cf-8145-e42a2017a923.png)
![Screen Shot 2022-05-20 at 4 14 35 PM](https://user-images.githubusercontent.com/10750442/169611422-d6fad79e-a780-40c2-9a00-6e670d411c58.png)

Reply:
Yup, fragment works fine.